client/gcp/gcp.go

```diff
@@ -0,0 +1,212 @@
+package gcp
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"github.com/bytebase/dbcost/client"
+)
+
+// Client is the client struct
+type Client struct {
+	httpClient http.Client
+}
+
+// NewClient return a client
+func NewClient() Client {
+	return Client{
+		httpClient: http.Client{},
+	}
+}
+
+const rdsServiceID = "9662-B51E-5089"
+
+// this apiKey should be blocked before merging
+const apiKey = "AIzaSyAj36rmGvQ_3kBqd6Lp3nSzPDo2_nD-sdM"
+
+// PriceInfoEndpoint is the endpoint for all Cloud SQL services on GCP
+// For more information, please refer to https://cloud.google.com/billing/v1/how-tos/catalog-api
+var PriceInfoEndpoint = fmt.Sprintf("https://cloudbilling.googleapis.com/v1/services/%s/skus?key=%s", rdsServiceID, apiKey)
+
+type unitPrice struct {
+	CurrencyCode string  `json:"currencyCode"`
+	Unit         string  `json:"units"`
+	Nano         float64 `json:"nanos"`
+}
+
+type tieredRate struct {
```

Comment:
Why are we missing public and private symbols? If these structs are not exposed, they should be all private. Otherwise, they should be all public.